streamalert_cli/athena/helpers.py

```diff
@@ -134,3 +147,98 @@ def unique_values_from_query(query_result):
         for row in query_result['ResultSet']['Rows'] for result in row['Data']
         for value in list(result.values())
     }
+
+def format_schema_tf(schema):
+    """Format schema for an Athena table for terraform.
+
+    Returns:
+        formatted_schema (list(tuple))
+    """
+    # Construct the main Athena Schema
+    formated_schema = []
+    for key_name in sorted(schema.keys()):
+        key_type = schema[key_name]
+        if isinstance(key_type, str):
+            formated_schema.append((key_name.lower(), key_type))
+        # Account for nested structs
+        elif isinstance(key_type, dict):
+            struct_schema = ','.join(
+                '{0}:{1}'.format(sub_key.lower(), key_type[sub_key])
+                for sub_key in sorted(key_type.keys())
+            )
+            formated_schema.append((key_name.lower(), 'struct<{}>'.format(struct_schema)))
+
+    return formated_schema
+
+def generate_alerts_table_schema():
+    """Generate the schema for alerts table in terraform by using a fake alert
+
+    Returns:
+        athena_schema (dict): Equivalent Athena schema used for generating create table statement
+    """
+    alert = Alert('temp_rule_name', {}, {})
```

Comment:
🤔

Reply:
I am sorry I am afraid I will leave this "jankiness" for now. I don't want to have too much changes in one PR.